test/escrow_spec.js

```diff
@@ -328,4 +327,84 @@ contract("Escrow", function() {
     assert.equal(balanceBeforeCreation, balanceAfterCancelation, "Invalid seller balance");
 
   });
+
+  describe("Rating a released Transaction", async() => {
+    let receipt, created, escrowId;
+
+    beforeEach(async() => {
+      const isPaused = await Escrow.methods.paused().call();
+      if (isPaused) {
+        receipt = await Escrow.methods.unpause().send({from: accounts[0]});
+      }
+      receipt = await Escrow.methods.create(accounts[1], expirationTime).send({from: accounts[0], value});
+      created = receipt.events.Created;
+      escrowId = created.returnValues.escrowId;
+      await Escrow.methods.release(escrowId).send({from: accounts[0]});
+    })
+
+    it("should not allow a score last less than 1", async() => {
+      try {
+        receipt = await Escrow.methods.rate_transaction(escrowId, 0).send({from: accounts[1]});
+        assert.fail('should have reverted: should not allow a score last less than 1');
+      } catch(error) {
+        TestUtils.assertJump(error);
+        assert.ok(error.message.indexOf('Rating needs to be at least 1') >= 0);
+      }
+    })
+
+    it("should not allow a score to be more than 5", async() => {
+      try {
+        receipt = await Escrow.methods.rate_transaction(escrowId, 6).send({from: accounts[1]});
+        assert.fail('should have reverted: should not allow a score to be more than 5');
+      } catch(error) {
+        TestUtils.assertJump(error);
+        assert.ok(error.message.indexOf('Rating needs to be at less than or equal to 5'))
+      }
+    })
+
+    for(let i=1; i<=5; i++) {
```

Comment:
I feel like doing 5 tests for that is a bit over kill. Just doing a test for 1 and 5 should be enough.